docs/api.rst

```diff
@@ -0,0 +1,264 @@
+
+   **LIST STREAMS, SHOW STREAMS**
+
+   :>json string statementText: The ksql statement for which the result is being returned.
+   :>json array  streams: List of streams.
+   :>json string streams[i].name: The name of the stream.
+   :>json string streams[i].topic: The topic backing the stream.
+   :>json string streams[i].format: The serialization format of the data in the stream. One of JSON, AVRO, or DELIMITED.
+
+   **LIST TABLES, SHOW TABLES**
+
+   :>json string statementText: The ksql statement for which the result is being returned.
+   :>json array  tables: List of tables.
+   :>json string tables[i].name: The name of the table.
+   :>json string tables[i].topic: The topic backing the table.
+   :>json string tables[i].format: The serialization format of the data in the table. One of JSON, AVRO, or DELIMITED.
+
+   **LIST QUERIES, SHOW QUERIES**
+
+   :>json string statementText: The ksql statement for which the result is being returned.
+   :>json array  queries: List of queries.
+   :>json string queries[i].queryString: The text of the statement that started the query.
+   :>json string queries[i].kafkaTopic: The topic that the query is writing into.
+   :>json string queries[i].id.id: The query id.
+
+   **LIST PROPERTIES, SHOW PROPERTIES**
+
+   :>json string statementText: The ksql statement for which the result is being returned.
+   :>json map    properties: The properties the ksql server runs queries with.
+   :>json string properties[<property-name>]: The value of the property named by property-name.
+
+   **DESCRIBE**
+
+   :>json string  statementText: The ksql statement for which the result is being returned.
+   :>json string  sourceDescription.name: The name of the stream or table.
+   :>json array   sourceDescription.readQueries: The queries reading from the stream or table.
+   :>json array   sourceDescription.writeQueries: The queries writing into the stream or table
+   :>json array   sourceDescription.schema: The schema of the stream or table as a list of column names and types.
+   :>json string  sourceDescription.schema[i].name: The name of the column.
+   :>json string  sourceDescription.schema[i].type: The data type of the column.
+   :>json string  sourceDescription.type: STREAM or TABLE
+   :>json string  sourceDescription.key: The name of the key column.
+   :>json string  sourceDescription.timestamp: The name of the timestamp column.
+   :>json string  sourceDescription.format: The serialization format of the data in the stream or table. One of JSON, AVRO, or DELIMITED.
+   :>json string  sourceDescription.topic: The topic backing the stream or table.
+   :>json boolean sourceDescription.extended: A boolean indicating whether this is an extended description.
+   :>json string  sourceDescription.statistics: A string containing statistics about production/consumption to/from the backing topic (extended only)
```

Comment:
`A string that contains statistics about production and consumption to and from the backing topic (extended only).`